src/MongoBulkDataMigration.ts

```diff
@@ -290,6 +295,22 @@ export default class MongoBulkDataMigration<TSchema extends Document>
       return { ok: status ? 1 : 0 } as any;
     }
 
+    if(this.options.noBackup) {
```

Comment:
@JulienBara-360 Tip: you can run `npm run prettier` to format in this project

Reply:
@pp0rtal 
Thanks :pray: 
Done here: https://github.com/360Learning/mongo-bulk-data-migration/pull/8/commits/52be004c339d1807927dd986c99d8a99dba4c959